hack/install/install-crictl.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Copyright 2018 The containerd Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+source $(dirname "${BASH_SOURCE[0]}")/utils.sh
+
+# Create a temporary GOPATH for crictl installation.
+TMPGOPATH=$(mktemp -d /tmp/cri-install-crictl.XXXX)
+GOPATH=${TMPGOPATH}
+
+#Install crictl
```

Comment:
since we've split this into it's own install.. and moved the from-vendor calls to each separate install.. let's also go ahead and move the CRITOOL_ variable assignments from utils.sh to here...

Reply:
Will do.